### ISSUE TITLE
Qubes 4.1 qubes-global-settings errors out and closes.

### ISSUE THREAD

**Issue by lyuts** [QubesOS/qubes-issues#7306]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### Qubes OS release

Qubes release 4.1 (R4.1)

### Brief summary

Qubes global settings raises an error and closes right away.

### Steps to reproduce
In terminal

$ qubes-global-settings

or via GUI

Qubes Manager -> System -> Global settings

### Expected behavior

Dialog with qubes global settings opens and allows changing settings.

### Actual behavior

Global setting dialog closes right away

```
Whoops. A critical error has occured. This is most likely a bug in Qubes Manager.

Exception: Cannot detect enabled ITL template update repositories
at line 11
of file /usr/bin/qubes-global-settings.
```

Details:
```
----
line: raise Exception(self.tr('Cannot detect enabled ITL template update '
func: __init_updates__
line no.: 513
file: /usr/lib/python3.8/site-packages/qubesmanager/global_settings.py
----
line: self.__init_updates__()
func: __init__
line no.: 88
file: /usr/lib/python3.8/site-packages/qubesmanager/global_settings.py
----
line: window = window_class(qt_app, qubes_app)
func: run_synchronous
line no.: 558
file: /usr/lib/python3.8/site-packages/qubesmanager/utils.py
----
line: utils.run_synchronous(GlobalSettingsWindow)
func: main
line no.: 666
file: /usr/lib/python3.8/site-packages/qubesmanager/global_settings.py
----
line: load_entry_point('qubesmanager==4.1.22', 'console_scripts', 'qubes-global-settings')()
func: <module>
line no.: 11
file: /usr/bin/qubes-global-settings

```

**Comment by andrewdavidwong** [comment]
> Exception: Cannot detect enabled ITL template update repositories

Are these repos actually disabled in your system? (Regardless, it should fail more gracefully, of course.)

**Comment by lyuts** [comment]
> Are these repos actually disabled in your system?

Yes they are. I'm judging based on the following:

```
$ qvm-template repolist
qubes-templates-itl  Qubes Templates repository  enabled
```

**Comment by DemiMarie** [comment]
> > Are these repos actually disabled in your system?
> 
> Yes they are. I'm judging based on the following:
> 
> ```
> $ qvm-template repolist
> qubes-templates-itl  Qubes Templates repository  enabled
> ```

They are enabled

**Comment by lyuts** [comment]
Looks like this invocation isn't showing ITL templates repos as enabled ones. Hopefully correctly reconstructed command returns this:

```
$ sudo /etc/qubes-rpc/qubes.repos.List 
updatesFedora 32 - x86_64 - Updatesenabled
updates-debuginfoFedora 32 - x86_64 - Updates - Debugdisabled
updates-sourceFedora 32 - Updates Sourcedisabled
fedoraFedora 32 - x86_64enabled
fedora-debuginfoFedora 32 - x86_64 - Debugdisabled
fedora-sourceFedora 32 - Sourcedisabled
qubes-dom0-currentQubes Dom0 Repository (updates)enabled
qubes-dom0-current-testingQubes Dom0 Repository (updates-testing)disabled
qubes-dom0-security-testingQubes Dom0 Repository (security-testing)disabled
qubes-dom0-unstableQubes Dom0 Repository (unstable)disabled
```

I think at this point I'm somewhat confused as to what is the right way of managing templates in 4.1. I suspect https://www.qubes-os.org/doc/templates/ is outdated?

**Comment by marmarek** [comment]
The fix is already at https://github.com/QubesOS/qubes-core-admin-linux/pull/87

**Comment by lyuts** [comment]
Ack. Thank you for update and fix. Any rough ETA when new package will land in repo?

**Comment by brendanhoar** [comment]
I am reporting the same issue.

**Comment by andrewdavidwong** [comment]
> Still seeing same error after the update. Update performed with 
> 
> ```
> 
> $ sudo qubes-dom0-update --console --show-output --enablerepo=qubes-dom0-current-testing
> 
> ```
> 
> 
> 
> ```
> 
> $ rpm -qa | grep qubes-manager
> 
> qubes-manager-4.1.23-1.fc32.noarch
> 
> ```

In https://github.com/QubesOS/qubes-issues/issues/7306#issuecomment-1050448305, @marmarek provided a link to the PR that fixes this. As you can see, that PR has not been merged yet, so I don't think you can get the fix just by updating (even from testing repos) yet.

**Comment by lyuts** [comment]
@andrewdavidwong You are totally right. Tracking multiple issues and got confused by one of the updates.

**Comment by tlaurion** [comment]
@marmarek I do not see bot posting here that it landed in stable repo even if 14 days ago

**Comment by marmarek** [comment]
Looks to be missing "Fixes" tag in the commit message.
